packages/wrangler/src/index.ts

```diff
@@ -253,6 +254,15 @@ export function createCLIParser(argv: string[]) {
 					process.env[key] = value;
 				}
 			}
+
+			// Write a session entry to the output file (if there is one).
+			writeOutput({
+				type: "wrangler-session",
+				version: 1,
+				wrangler_version: wranglerVersion,
+				command_line_args: argv,
+			});
```

Comment:
Great default entry with args for context! 👏 

Can you also add the log file path as well

Reply:
Added : https://github.com/cloudflare/workers-sdk/pull/6383/commits/34ac024023326edd5b132fb7981eab968c4864eb